### PR TITLE
Dev.ej/web app docs

### PR DESCRIPTION
**Pull request by joanise**
### PR Goal? 

Add documentation for the ReadAlong Studio web app and web editor

### Fixes? 

This is part of https://github.com/ReadAlongs/Studio-Web/issues/298

### Feedback sought? 

Read the documentation I wrote and let me know how you like it, if it should be improved, etc.

### Priority? 

this week if possible

### Tests added? 

n/a

### How to test? 

Serve locally:
```
git checkout dev.ej/web-app-docs
pip install -r docs/requirements.txt
mkdocs serve
```
and navigate to http://127.0.0.1:8000/web-app/

Or check it out pre-built at https://joanise.github.io/ReadAlong-Studio/dev/web-app/

### Confidence? 

medium

### Version change? 

no

### CODE REVIEW

File: docs/outputs.md [hunk 1]
```diff
@@ -25,12 +38,13 @@ Below is an example of a minimal implementation in a basic standalone html page.
         <!-- Here is how you declare the Web Component -->
         <read-along href="assets/sample.readalong" audio="assets/sample.wav"></read-along>
     </body>
+
     <!-- The last step needed is to import the package -->
     <script type="module" src='https://unpkg.com/@readalongs/web-component@^1.4.0/dist/web-component/web-component.esm.js'</script>
```
Comment (deltork) [inline docs/outputs.md:43]: 1.4.x

**Review by deltork — APPROVED** [review]
Looks good!

File: docs/outputs.md (hunk 1 above)
Comment (joanise) [reply docs/outputs.md:43]: `^1.4.0` is the same as`1.4.x`, no?

**Comment by marctessier** [comment]
checked it out. Looks good Eric.

**Pull request merged**